pyproject.toml

```diff
@@ -86,6 +86,10 @@ make-summary-multi-line = true
 wrap-summaries = 79
 wrap-descriptions = 79
 
+[tool.mypy]
+exclude = ["pygmt/tests/"]
+ignore_missing_imports = true
```

Comment:
Some packages like geopandas don't provide type hints and mypy reports following errors:
```
Cannot find implementation or library stub for module named "geopandas"  [import-not-found]
```
Ignore these errors following https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports.